tests/test_profiler.py

```diff
@@ -26,6 +26,9 @@ def test_simple_profiler():
     np.testing.assert_allclose(p.recorded_durations["b"], [2], rtol=0.2)
     np.testing.assert_allclose(p.recorded_durations["c"], [1], rtol=0.2)
 
+    # ensure this doesn't throw any errors
+    description = p.describe()
```

Comment:
I do not see `description` to be used...

Reply:
yes i’m just making sure that it can successfully call the method without throwing an error. might be worth moving that line to a new test case `test_simple_profiler_describe()`

Reply:
you may just drop output `_ = p.describe()` or just `p.describe()`
I would say that single letter variable is not good practice... :]

Reply:
agreed, i'll clean the tests up a bit